script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1157,87 +1236,98 @@ I call out to her.
 [sha:21d1c39fbe6693466a9b3518af360b3085e70524]{
 -- Page 459, Offset 34614.
 -- 「……ずるい……遠野くんは、ずる、い……！
-"...That's not fair... that's not fair, Tohno-kun...!
+"That's not fair... that's not fair, Tohno-kun...!
```

Comment:
How could you on next line, so might keep not fair here just for variety

Reply:
Yeah, I did notice. Since it was the same JP word, I'm not sure if it "variety" was a concern here, honestly. I think breaking down crying isn't exactly a verbose act so to speak. Might flow well into the next anyway?

I just think the whole fairness thing or what not is just very much a JPism around this word when it's actually like: "how could I ever not be affected by that?". But, yeah, it's just a hard nuance. Maybe just "how" on it's own is better?